mlflow/pyfunc/__init__.py

```diff
@@ -205,6 +205,7 @@
 from mlflow.models import Model
 from mlflow.pyfunc.model import PythonModel, PythonModelContext,\
     DEFAULT_CONDA_ENV
+from mlflow.tracking.artifact_utils import _download_artifact_from_uri
```

Comment:
are we pretty comfortable that the `artifacts` arg in `save_model`, `log_model` work okay with `runs:/` URI? it looks like we end up using `_download_artifact_from_uri` on them so it seems okay, but you have more context.

Reply:
Good point! I updated `test_model_export_with_class_and_artifacts#test_model_log_load` to reference an artifact with a `runs:/` URI!